extensions/wikia/PortableInfobox/services/PortableInfoboxMobileRenderService.class.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+use Wikia\PortableInfobox\Helpers\PortableInfoboxRenderServiceHelper;
+
+class PortableInfoboxMobileRenderService extends PortableInfoboxRenderService {
+	const MEDIA_CONTEXT_INFOBOX_HERO_IMAGE = 'infobox-hero-image';
+	const MEDIA_CONTEXT_INFOBOX = 'infobox';
+
+	protected $imagesWidth = PortableInfoboxRenderServiceHelper::MOBILE_THUMBNAIL_WIDTH;
+
+	/**
+	 * renders infobox
+	 *
+	 * @param array $infoboxdata
+	 *
+	 * @param $theme
+	 * @param $layout
+	 * @param $accentColor
+	 * @param $accentColorText
+	 * @return string - infobox HTML
+	 */
+	public function renderInfobox( array $infoboxdata, $theme, $layout, $accentColor, $accentColorText ) {
```

Comment:
Side note:
In this case I don't have strong opinion whether we should convert parameters to shorter number or not. Although I believe using objects instead of arrays makes more sense for passing configuration data. Class clearly states what is structure of that config and can provide default values.